src/main/java/org/jabref/logic/util/StandardFileType.java

```diff
@@ -52,6 +52,11 @@
     }
 
     public static FileType newFileType(String... extensions) {
+        for (int i = 0; i < extensions.length; i++) {
```

Comment:
Could you please explain to me why this is necessary?
You remove the dot from extension?

Reply:
No problem!  Since I am new, if you have another idea of how to implement this, let me know.

If you look in lines 97-100 in `org.jabref.preferences.CustomExportList` on the master branch, you will see that there is a check for the `.` while processing extensions passed into the class, usually as object `Globals.prefs.customImports` (also removed in PR branch).  In the `master` branch, this is because the TemplateExporter constructor takes a `FileType` without a `.`  Since this is text processing, and could be relevant to any class that wants to use `StandardFileType`, I decided to put it into logic.  This way, that check is done inside the `StandardFileType`.  It doesn't impose any new requirements, so it should not break anything else.  That said, I created a new constructor in `TemplateExporter` and could put it there instead.

As a side note, I forgot to reference the new constructor (with fewer parameters), and have pushed changes referencing it.